### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by pombredanne** [Yacoby/suffix-tree-unicode#1]
What would be the license for this fine code?

**Comment by Yacoby** [comment]
I don't actually know as the code I built on didn't have a licence that I could find.

From my perspective I am happy for my changes to be used under the MIT licence, but I can't speak for the other authors.

**Comment by pombredanne** [comment]
fair enough ... I sent an email to @mailund who is the author of the original implementation.... He may be able to chime in here too! ... I understand that the original code was based on Perl-licensed code... but these are older versions. Note sure what would be Thomas license otherwise for the latests

**Comment by mailund** [comment]
If there isn't a license in the tar ball I'm happy to release it under either perl or MIT. It is not really that important to me.

**Comment by pombredanne** [comment]
@mailund Thanks for following up! So The license is a choice of Perl or MIT then. Or just MIT? (it would be compatible GPL and Perl also)

**Comment by mailund** [comment]
Pick one. I'm happy with making it available in either
